test/evaluation/test_nDCG.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict, Sequence
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+from pytest import mark, skip
+
+from rsdiv.evaluation import RankingMetrics
+
+
+class TestNDCG:
+    rng = np.random.default_rng(42)
+
+    @mark.parametrize("item_size", [10, 100, 500])
+    @mark.parametrize("relevant_size", [10, 100, 500])
+    @mark.parametrize("position", [10, 100, 500])
+    @mark.parametrize("exponential", [True, False])
+    def test_equality(
+        self, item_size: int, relevant_size: int, position: int, exponential: bool
+    ) -> None:
+        if position > min(item_size, relevant_size) or relevant_size > item_size:
+            skip()
+        scores = self.rng.random(item_size)
+        sorted_indices = np.argsort(scores)
+        relevant_id = sorted_indices[: -relevant_size - 1 : -1]
+        assert_allclose(
+            RankingMetrics.nDCG(
```

Comment:
Value of type variable "T" of "nDCG" of "RankingMetrics" cannot be "Union[SupportsIndex, Tuple[SupportsIndex, ...]]"